content/docs/learning-guide/tutorials/multiplayer/first-multiplayer-component.md

```diff
@@ -747,12 +749,22 @@ Now that we have created a data flow from the server to clients using network pr
     ```c++
     void MyFirstNetworkComponent::OnUpTimeChanged(double uptime)
     {
-        AZLOG("MyFirstNetworkComponent", "client = %f", uptime);
+        AZ_Printf("MyFirstNetworkComponent", "client = %f\n", uptime);
         static_cast<MyFirstNetworkComponentController*>(GetController())->SendConfirmUptime(uptime);
     }
     ```
 
-    ![Attaching Network Component to an Autonomous Entity](/images/user-guide/multiplayer/add_myfirstnetworkcomponent_to_player_prefab.png)
+1. Finally, add a check to `MyFirstNetworkComponentController::OnActivate` to ensure it is only connecting to the TickBus while it's executing on the server (e.g., is not the autonomous player client, which will also have a controller):
```

Comment:
```suggestion
1. Finally, add a check to `MyFirstNetworkComponentController::OnActivate` to ensure `MyFirstNetworkComponentController` is only connecting to the `TickBus` while it's executing on the server. For example, it's connecting to the server rather than the autonomous player client, which also has a controller):
```
- Can you verify that "it" refers to `MyFirstNetworkComponentController`?
- I expanded your example at the end, since we try to avoid using Latin abbreviations such as "e.g.". Want to make sure with you that the intent of the example there hasn't changed.